applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -198,9 +198,9 @@ impl wallet_server::Wallet for WalletGrpcServer {
     async fn identify(&self, _: Request<GetIdentityRequest>) -> Result<Response<GetIdentityResponse>, Status> {
         let identity = self.wallet.comms.node_identity();
         Ok(Response::new(GetIdentityResponse {
-            public_key: identity.public_key().to_string().into_bytes(),
+            public_key: identity.public_key().to_string(),
```

Comment:
```suggestion
            public_key: Vec::from(identity.public_key().as_bytes())
```

Reply:
Thanks ! Why not just `identity.public_key().to_vec()` ?